scripts/dependencies/lima.ts

```diff
@@ -56,16 +56,10 @@ export class Lima implements Dependency, GitHubDependency {
     const baseUrl = `https://github.com/${ this.githubOwner }/${ this.githubRepo }/releases/download`;
     let platform: string = context.platform;
 
-    // If the name of the forward compatible limactl binary changes or the
-    // binary is built with a newer version of xcode please update, the
-    // fwdCompatLimactlBin or fwdCompatLimactlDarwinVer
-    // constants in backend/lima.ts.
-    const fwdCompatLimactlBin = 'limactl.ventura';
-
     if (platform === 'darwin') {
-      platform = 'macos-11.amd64';
+      platform = 'macos-13.amd64';
```

Comment:
Optional, would it make sense to go in a variable?

Reply:
Not sure what should go into a variable. I've turned the 4 lines into a single line with expression interpolations, but that's all I can think off.

The hard-coded `macos-13` is a bit annoying, but I don't know how we can parameterize it; it depends on a GitHub Action workflow in a different repo...